wcfsetup/install/files/lib/data/user/group/UserGroup.class.php

```diff
@@ -56,6 +56,12 @@ class UserGroup extends DatabaseObject implements ITitledObject {
 	 */
 	const OTHER = 4;
 	
+	/**
+	 * the owner group is always an administrator group
+	 * @var int
+	 */
+	const OWNER = 5;
```

Comment:
`5` to `7` are in use by `com.woltlab.wcf.moderatedUserGroup`.

Reply:
Fixed it.